plume-models/src/follows.rs

```diff
@@ -35,7 +35,14 @@ pub struct NewFollow {
 }
 
 impl Follow {
-    insert!(follows, NewFollow);
+    insert!(follows, NewFollow, |inserted, conn| {
+        if inserted.ap_url.is_empty() {
+            inserted.ap_url = ap_url(&format!("{}/follows/{}", *BASE_URL, inserted.id));
```

Comment:
this route should really get implemented one day

Reply:
Yes, I think we already have an issue for it…